lib/minidoc/duplicate_key.rb

```diff
@@ -0,0 +1,13 @@
+class Minidoc::DuplicateKey < Mongo::OperationFailure
+  DUPLICATE_KEY_ERROR_CODE = 11000
```

Comment:
Couldn't find a good, canonical reference for this, but did look. Mostly gotten this through experience with the error.